.github/workflows/python-package.yml

```diff
@@ -27,7 +26,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8 pytest numpy h5py
+        python -m pip install flake8 pytest numpy h5py coverage
```

Comment:
I recommend going all-in on Python 3.  This looks like it will produce a mix of Py2 and Py3 in your CI environment, which sounds like a recipe for trouble.